src/styles/_mixins.scss

```diff
@@ -0,0 +1,24 @@
+@mixin error {
+    font-family: Helvetica;
+    font-size: 12px;
+    line-height: 18px;
+    letter-spacing: 0.09px;
+}
+
+@mixin text {
+    font-size: 14px;
+    line-height: 18px;
+    letter-spacing: 0.1px;
+}
+
+@mixin form-text-content {
+    @include text;
+
+    font-family: Helvetica;
+}
+
+@mixin input-text {
+    @include text;
+
+    font-family: "Source Sans Pro";
```

Comment:
why don't just leave the default one? for me it looks like this:
![image](https://user-images.githubusercontent.com/1554577/56568678-2b887480-65c0-11e9-8bdc-ec9812213786.png)

Reply:
Design says so